development/extensions/skeleton_extension.rst

```diff
@@ -30,17 +30,17 @@ installed into a phpBB board just the same as any other.
 
 .. note::
 
-    The Skeleton Extension is a tool for extension developers. As such,
-    it is intended to be installed in a phpBB board used for
-    offline/local development of extensions. It is not necessarily
-    intended to be installed on live/production web sites.
+    The Skeleton Extension is intended to be installed in a phpBB
+    board used for offline/local development of extensions. It is
+    not necessarily intended to be installed on live/production web
```

Comment:
There really isn't anything dangerous about having this extension installed so if someone installed it on their live board the worst that will happen is that they will have a really long link in their nav bar.